### PR TITLE
Email templates

### PR DESCRIPTION
**Pull request by Singletoned**

### CODE REVIEW

**Review by rottitime — CHANGES_REQUESTED** [review]
@Singletoned I seem to be getting the following errors only when I switch or rebase to this branch

`docker compose up api`

```
ellandi-api-1  | Traceback (most recent call last):
ellandi-api-1  |   File "manage.py", line 22, in <module>
ellandi-api-1  |     main()
ellandi-api-1  |   File "manage.py", line 18, in main
ellandi-api-1  |     execute_from_command_line(sys.argv)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
ellandi-api-1  |     utility.execute()
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 363, in execute
ellandi-api-1  |     settings.INSTALLED_APPS
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/django/conf/__init__.py", line 82, in __getattr__
ellandi-api-1  |     self._setup(name)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/django/conf/__init__.py", line 69, in _setup
ellandi-api-1  |     self._wrapped = Settings(settings_module)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/django/conf/__init__.py", line 170, in __init__
ellandi-api-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
ellandi-api-1  |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
ellandi-api-1  |     return _bootstrap._gcd_import(name[level:], package, level)
ellandi-api-1  |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
ellandi-api-1  |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
ellandi-api-1  |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
ellandi-api-1  |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
ellandi-api-1  |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
ellandi-api-1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ellandi-api-1  |   File "/app/api/ellandi/settings.py", line 54, in <module>
ellandi-api-1  |     CORS_ALLOWED_ORIGINS = (HOST_URL, HOST_MAP[HOST_URL])
ellandi-api-1  | KeyError: 'localhost:8001'
ellandi-api-1 exited with code 1

**Comment by Singletoned** [comment]
I've fixed that now

**Review by rottitime — CHANGES_REQUESTED** [review]
@Singletoned @nmenezes0 I am getting the following traceback error for email verification:

Request: `http://localhost:8000/user/63e62a47-9d0d-4e18-9511-e5a04ce8db1c/verify/bb0zsk-3804b4b5b62858c316a1798b3e52ad67/`

```
ellandi-api-1  | Invalid token: /user/63e62a47-9d0d-4e18-9511-e5a04ce8db1c/verify/bb0zsk-3804b4b5b62858c316a1798b3e52ad67/
ellandi-api-1  | Traceback (most recent call last):
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
ellandi-api-1  |     response = get_response(request)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
ellandi-api-1  |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/sentry_sdk/integrations/django/views.py", line 67, in sentry_wrapped_callback
ellandi-api-1  |     return callback(request, *args, **kwargs)
ellandi-api-1  |   File "/usr/local/lib/python3.8/contextlib.py", line 75, in inner
ellandi-api-1  |     return func(*args, **kwds)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
ellandi-api-1  |     return view_func(*args, **kwargs)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 70, in view
ellandi-api-1  |     return self.dispatch(request, *args, **kwargs)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
ellandi-api-1  |     response = self.handle_exception(exc)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
ellandi-api-1  |     self.raise_uncaught_exception(exc)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
ellandi-api-1  |     raise exc
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
ellandi-api-1  |     response = handler(request, *args, **kwargs)
ellandi-api-1  |   File "/usr/local/lib/python3.8/site-packages/rest_framework/decorators.py", line 50, in handler
ellandi-api-1  |     return func(*args, **kwargs)
ellandi-api-1  |   File "/app/api/ellandi/verification.py", line 72, in verification_view
ellandi-api-1  |     raise BadRequest("Invalid token")
ellandi-api-1  | django.core.exceptions.BadRequest: Invalid token

File: api/ellandi/templates/email/verification.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Your email {{user.email}} address was registered with Ellandi
```
Comment (nmenezes0) [inline api/ellandi/templates/email/verification.txt:1]: Should we be calling it "Ellandi"?
Or maybe you're waiting on a polished version of the text from Heather anyway.

Comment (Singletoned) [reply api/ellandi/templates/email/verification.txt:1]: Yeah, I have no idea what we're calliing it.  It's just placeholder text

**Comment by nmenezes0** [comment]
I assume it's expected that the URLs in the email don't work yet?
eg `http://localhost:3000/signin/forgotten-password/reset?code=bb2lbb-c0c12d9d01c61b353b50dcc3b0d2f5e8&user_id=33425117-3a02-4bd2-94fb-367f062a736d`

(Have found URLs for API eg `http://localhost:8000/user/33425117-3a02-4bd2-94fb-367f062a736d/password-reset/bb2lbb-c0c12d9d01c61b353b50dcc3b0d2f5e8/` are fine.)

**Comment by nmenezes0** [comment]
> 

I haven't been getting these errors (though Ed pushed code since you made comments, so maybe issue was fixed).

**Review by nmenezes0 — CHANGES_REQUESTED** [review]
Happy with changes to Ellandi - but why are the organogram tests failing?

Also left a couple of comments in PR.

**Comment by Singletoned** [comment]
> I assume it's expected that the URLs in the email don't work yet? eg `http://localhost:3000/signin/forgotten-password/reset?code=bb2lbb-c0c12d9d01c61b353b50dcc3b0d2f5e8&user_id=33425117-3a02-4bd2-94fb-367f062a736d`

Yeah, we need the frontend as well, before they work

**Comment by Singletoned** [comment]
> Happy with changes to Ellandi - but why are the organogram tests failing?

No idea.  It's really weird.  They are passing locally.  I'll have a look into it

**Comment by rottitime** [comment]
> > I assume it's expected that the URLs in the email don't work yet? eg `http://localhost:3000/signin/forgotten-password/reset?code=bb2lbb-c0c12d9d01c61b353b50dcc3b0d2f5e8&user_id=33425117-3a02-4bd2-94fb-367f062a736d`
> 
> Yeah, we need the frontend as well, before they work

The FE is all done. I have deployed to demo
https://ellandi-web-demo.london.cloudapps.digital/

**Review by rottitime — APPROVED** [review]
@Singletoned Can this get merged in please so I can move various tickets for testing.

**Review by nmenezes0 — APPROVED** [review]

**Pull request merged**